exosphere-shared/templates/create-app/application-go.yml

```diff
@@ -0,0 +1,11 @@
+name: {{.AppName}}
```

Comment:
Using explicit templates in files does not work in Go. Let's use strings in files instead.